### ISSUE TITLE
jump-links subsections aren't reachable by keyboard

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2411]
## Description of the issue

jump-links subsections aren't reachable by keyboard

### Impacted component(s)

- `pf-jumplinks`

### Steps to reproduce

1. Go to [jump-links subsection demo](https://patternflyelements.org/components/jump-links/#expandable-vertical-with-subsection)
2. Focus on **Jump to Section**.
3. Click <kbd>Enter</kbd>.
4. Use arrow keys to focus on  any "subsection".

### Expected behavior

Subsections should be reachable by arrow keys.
